### PR TITLE
Release 2.2

### PR DESCRIPTION
**Pull request by abertschi**
- drop support for legacy endpoint wrapper
- fix username/ password authentication

### CODE REVIEW

**Comment by davidoesch** [comment]
me sad , very sad

**Comment by abertschi** [comment]
Maybe I should add some more comments:
- I did solely drop support for https://postcardcreator.post.ch/rest/2.2 as the endpoints do not work anymore (https://github.com/abertschi/postcard_creator_wrapper/blob/c09f07f00d04689c17a56b69341aa657b8673d6f/postcard_creator/postcard_creator_legacy.py#L34) 
- These are technical notes. From a user perspective, nothing changed. You can still use legacy username+password credentials without swissid, or use swissid respectively. Both authentication methods are now served with PostcardCreatorSwissId (https://github.com/abertschi/postcard_creator_wrapper/blob/c09f07f00d04689c17a56b69341aa657b8673d6f/postcard_creator/postcard_creator_swissid.py#L35 ) which uses a token that is obtained with a code+code_verifier.

tldr: If you do not rely on get_user_info, get_billing_saldo, get_quota, nothing changed. If you rely on these methods and used authentication method=legacy, they now return data from PostcardCreatorSwissId and their output is slightly different.

**Comment by abertschi** [comment]
postcards is now updated to use version 2.2 of this project. https://github.com/abertschi/postcards

**Comment by davidoesch** [comment]
just tested...works like a charm! me: Happy.

thanks a lot for keeping legacy working... amazing work
